### PR TITLE
Add transaction wrapper for creating transactions

### PR DESCRIPTION
**Pull request by saltz**

### CODE REVIEW

File: src/__tests__/transaction.test.ts [hunk 1]
```diff
@@ -10,3 +11,15 @@ describe("Transaction object", () => {
         expect(transaction).toBeInstanceOf(Transaction);
     });
 });
+
+describe("Transaction wrapper object", () => {
+    it("Should instantiate with valid parameters", () => {
+        const transactionWrapper = new TransactionWrapper(
+            new Transaction(TransactionType.COIN, "RICIPIENT_KEY", "SENDER_KEY", 1, new Date()),
```
Comment (timofudala) [inline src/__tests__/transaction.test.ts]: Please fix the typo in `RICIPIENT_KEY`.

File: src/transactionWrapper.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Transaction } from "./transaction";
+
```
Comment (timofudala) [inline src/transactionWrapper.ts:2]: Please add some proper docs regarding why we need a wrapper here.

**Review by timofudala — CHANGES_REQUESTED** [review]

**Review by timofudala — APPROVED** [review]

File: src/transactionWrapper.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { Transaction } from "./transaction";
+/**
```
Comment (RikvanSpreuwel) [inline src/transactionWrapper.ts:3]: Consider adding a new line between import and the tsdoc.

```suggestion

/**
```

**Review by RikvanSpreuwel — CHANGES_REQUESTED** [review]
See comment

**Review by RikvanSpreuwel — APPROVED** [review]

File: src/__tests__/transaction.test.ts [hunk 4]
```diff
@@ -10,3 +11,15 @@ describe("Transaction object", () => {
         expect(transaction).toBeInstanceOf(Transaction);
     });
 });
+
+describe("Transaction wrapper object", () => {
+    it("Should instantiate with valid parameters", () => {
+        const transactionWrapper = new TransactionWrapper(
+            new Transaction(TransactionType.COIN, "RECIPIENT_KEY", "SENDER_KEY", 1, new Date()),
```
Comment (ronaldvandenbroek) [inline src/__tests__/transaction.test.ts:18]: What does this 1 signify here? Might be better to create a constant for this number.

**Pull request merged**